tests/test_utils.py

```diff
@@ -267,14 +268,17 @@ def test_compute_class_weights(self):
 class TestGradientAccumulation(unittest.TestCase):
 
     @tf.function
-    def test_gradient_accumulating_optimizer(self):
+    def body_of_test_gradient_accumulating_optimizer(self, opt):
         with tf.Graph().as_default():
             loss = tf.compat.v1.get_variable("loss", shape=1)
             lr = 0.1
-            opt = get_grad_accumulation_optimizer(tf.keras.optimizers.SGD, 2)(lr)
+            opt = get_grad_accumulation_optimizer(opt, 2)(lr)
             global_step = tf.compat.v1.train.get_or_create_global_step()
-            with tf.control_dependencies([opt.minimize(lambda: tf.abs(loss), [loss])]):
-                train_op = global_step.assign_add(1)
+            if isinstance(opt, tf.keras.optimizers.Optimizer):
```

Comment:
So seems like we're still keeping partial support for keras optimizers -- just defaulting to ones that are compatible w/ multi-gpu distribution strategies?